diagnostics/diag.go

```diff
@@ -98,7 +98,13 @@ func (di *DiagInfo) Marshal() []byte {
 }
 
 func (d *Diagnostics) getPeers() []peer.ID {
-	return d.host.Network().Peers()
```

Comment:
so, this was returning dupes?? sure? or are any of these offsets being used as indices somewhere?